frontend/src/static/js/components/webstatus-feature-wpt-progress-chart-panel.ts

```diff
@@ -34,8 +33,16 @@ import {
 
 @customElement('webstatus-feature-wpt-progress-chart-panel')
 export class WebstatusFeatureWPTProgressChartPanel extends WebstatusLineChartPanel {
```

Comment:
Thoughts on making a new abstract WebstatusLineChartTabbedPanel class and have this class extend class extend that instead of the current `WebstatusLineChartPanel` ? You have additional abstract properties. Then if we could override render() with to use getTabs() instead of putting it in WebstatusLineChartPanel

Reply:
This sounds like a good idea - I'll update this PR with that approach 🙂